src/meta/src/hummock/manager/mod.rs

```diff
@@ -1273,10 +1283,20 @@ where
                     compact_task.set_task_status(TaskStatus::InputOutdatedCanceled);
                     false
                 } else {
-                    assert!(current_version
+                    let group = current_version
                         .levels
-                        .contains_key(&compact_task.compaction_group_id));
-                    true
+                        .get(&compact_task.compaction_group_id)
+                        .unwrap();
```

Comment:
can be none

Reply:
No. It must exist. Because we have check whether it exist before.